scripts/generateAssetData/baseAssets.ts

```diff
@@ -153,7 +153,7 @@ export const polygon: AssetWithNetworkName = {
   symbol: 'MATIC',
   precision: 18,
   color: '#8f00ff',
-  icon: 'https://assets.coingecko.com/coins/images/4713/large/matic-token-icon.png?1624446912',
+  icon: 'https://raw.githubusercontent.com/trustwallet/assets/b7a5f12d893fcf58e0eb1dd64478f076857b720b/blockchains/polygon/info/logo.png',
```

Comment:
That looks weird but that's correct, both variants are valid https://polygontechnology.notion.site/Polygon-Brand-Resources-External-2cd18ae436584e98a6c5aae56db73058?p=aa481b968f2e46c0a83f9da751e0aa17&pm=c